### PR TITLE
Templates - homework make_unique

### PR DESCRIPTION
**Pull request by niemarcin**

### CODE REVIEW

**Review by WBigaj — APPROVED** [review]
I have checked your code and it looks good 👍

File: module1/homework/make_unique/make_unique-ut.cpp [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    TestStruct() = default;
+    TestStruct(int i_val, double d_val)
+        : structValInt_(i_val), structValDbl_(d_val) {}
+
+    int structValInt_{};
+    double structValDbl_{};
+};
+
+struct LRvalTestStruct {
+    LRvalTestStruct(int val = testValInt_L)
```
Comment (ziobron) [inline module1/homework/make_unique/make_unique-ut.cpp:24]:
```suggestion
    explicit LRvalTestStruct(int val = testValInt_L)
```

File: module1/homework/make_unique/make_unique-ut.cpp [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+TEST(MakeUniqueTest, ShouldMakeUniqueForLvalTestStruct) {
+    auto uniquePtr = coders::make_unique<LRvalTestStruct>();
+    ASSERT_EQ(uniquePtr->testValue_, testValInt_L);
+}
+
+TEST(MakeUniqueTest, ShouldMakeUniqueForRvalTestStruct) {
+    LRvalTestStruct temp;
+    auto uniquePtr = coders::make_unique<LRvalTestStruct>(std::move(temp));
+    ASSERT_EQ(uniquePtr->testValue_, testValInt_R);
+}
```
Comment (ziobron) [inline module1/homework/make_unique/make_unique-ut.cpp:110]: what about copy constructor? you have tests for move and default constructor.

**Review by ziobron — APPROVED** [review]

**Comment by ziobron** [comment]
{ "requirement_ids": [1119, 4443], "points": [3, 40] }

**Pull request closed without merging**